taxi/admin.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+# Register your models here.
+
+
+@admin.register(Driver)
+class DriverAdmin(admin.ModelAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + (("Additional info", {"fields": ("license_number",)}),)
+    add_fieldsets = UserAdmin.add_fieldsets + (("Additional info", {"fields": ("license_number",)}),)
+
```

Comment:
I think this class should be inherited from UserAdmin, because Drivers in models.py is inherited from AbstractUser and anyway you use here ```UserAdmin.list_display```